appservice/.vscode/launch.json

```diff
@@ -8,18 +8,16 @@
             "runtimeExecutable": "${execPath}",
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}/test/extension",
-                "--extensionTestsPath=${workspaceFolder}/out/test"
+                "--extensionTestsPath=${workspaceFolder}/out/test/index"
             ],
-            "stopOnEntry": false,
-            "sourceMaps": true,
             "outFiles": [
                 "${workspaceFolder}/out/test/**/*.js"
             ],
-            "preLaunchTask": "npm: compile",
+            "preLaunchTask": "${defaultBuildTask}",
```

Comment:
Is `defaultBuildTask` pretest by default?  I didn't see where this gets defined anywhere.

Reply:
It's set here: https://github.com/microsoft/vscode-azureappservice/blob/master/.vscode/tasks.json#L9

This is one of the things that the new hello-world extensions from VS Code do, so I copied them. I guess it's a tiny bit better in case you rename the build task, but mostly it doesn't make a difference